src/routes/safe/components/Transactions/TxList/hooks/usePagedQueuedTransactions.ts

```diff
@@ -17,6 +17,7 @@ type PagedQueuedTransactions = {
 }
 
 export const usePagedQueuedTransactions = (): PagedQueuedTransactions => {
+  // get next & queued transactions from Redux
```

Comment:
This is self-explanatory. I'd remove it from the commit to reduce the size.

Reply:
removed